cmd/rpcdaemon/commands/eth_receipts.go

```diff
@@ -40,12 +40,10 @@ func (api *BaseAPI) getReceipts(ctx context.Context, tx kv.Tx, chainConfig *chai
 	// FIXME disabling the cached receipt path to avoid wiring L1 Fee stuff for
```

Comment:
Is this fixed? Will we store the receipts correctly for post-bedrock blocks? (If not, then this change will break e2e)

Reply:
I don't know. Is it fixed on the boba-develop? The receipt part works on my branch, but I don't know if it works in the boba-develop.

Reply:
We'll need to check I guess, but I assume this will break the e2e tests.  If you're interested, do you want to see about getting bedrock to populate these receipts fields and get the receipt serialized into the DB?

Reply:
The way that I did (I transform the big.Int to string so that we can encode it into receipt) is for the legacy block. For the new block, these special fields can be derived from the first deposit transaction which means that these fields are not in the receipt table (It's in the table, but they are 0 value. The cbor package can't encode big type).

We still have to encode these field for the legacy receipts, so this `I transform the big.Int to string so that we can encode it into receipt` should be pushed to boba-develop.

https://github.com/testinprod-io/op-erigon/blob/b1674c208a9748d32c3547fbb96c323ee9fa1b2d/core/types/receipt.go#L521